saleor/plugins/webhook/shipping.py

```diff
@@ -23,14 +24,33 @@
 logger = logging.getLogger(__name__)
 
 
-def to_shipping_app_id(app_id: int, shipping_method_id: str) -> "str":
+def to_shipping_app_id(app: "App", shipping_method_id: str) -> "str":
+    app_identifier = app.identifier or app.id
     return base64.b64encode(
-        str.encode(f"{APP_ID_PREFIX}:{app_id}:{shipping_method_id}")
+        str.encode(f"{APP_ID_PREFIX}:{app_identifier}:{shipping_method_id}")
     ).decode("utf-8")
 
 
+def to_new_shipping_app_id(shipping_app_id: str):
```

Comment:
Maybe something without `_new_`? In few months we won't know why something is `new`. 
Something like: `convert_to_app_id_with_identifier`?